cypress/integration/homework/calculator.spec.js

```diff
@@ -0,0 +1,90 @@
+import Homepage from '../../page-objects/homepage.js'
+import { timeout } from 'async';
+/// <reference types="cypress" />
+
```

Comment:
/// <reference types="cypress" />

    let selectBuild = ["Prototype", "1", "2", "3", "4", "5", "6", "7", "8", "9"];

// mačiau visas pasikartojančias operacijas išsikėlei kaip page objects, labai gerai. Dar pvz kaip galima supaprastinti testus, tai jas aprašyti kaip funkcijas

    function setFirstDigit(value) {
        cy.get("#number1Field").clear().type(value)
    }
    
    function setSecondDigit(value) {
        cy.get("#number2Field").clear().type(value)
    }
    
    function setOperation(value) {
        cy.get("#selectOperationDropdown").select(value)
    }
    
    for (let i = 0; i < selectBuild.length; i++){
        describe('Testing Build '+ selectBuild[i], () => {
            beforeEach("Execute before each test",() => {
                cy.visit("https://testsheepnz.github.io/BasicCalculator")
                cy.get("#selectBuild").select(selectBuild[i])
             })
            
    //testas su iškeltom funkcijom atrodytų maždaug taip
            
        it("Checking calculating values on addition", () => {
            setFirstDigit(2)
            setSecondDigit(3)
            setOperation("Multiply")
                ...
        })
            
    //arba galima operacijų nerašyti atskirai kaip funkcijų :) 
            
        it("Checking calculating values on subtraction", () => {
            cy.get("#number1Field").clear()
            cy.get("#number1Field").type("3")
            cy.get("#number2Field").clear()
            cy.get("#number2Field").type("2")
            cy.get("#calculateButton").click("Subtract")
            cy.get("#numberAnswerField").should("have.value", "1")
            cy.get("#clearButton").click()
        })
            
     // ir taip kartoti per visus testavimo scenarijus       
            ...
        })
    }